src/Controllers/MyRadio/login.php

```diff
@@ -16,7 +16,7 @@
  */
 
 if (isset($_SESSION['memberid'])) {
-    URLUtils::redirect(Config::$base_url);
+    URLUtils::redirectWithMessage("/", "", "It seems you were trying to login while still being logged in, if that is not what happened, please contact the Computing Team.");
```

Comment:
`"/", "",`
Doesn't really look right... Does this actually go to / or to /myradio/ or what is configured?